### PR TITLE
Fixed https://github.com/nim-lang/nimble/issues/305

### LINKED ISSUE #305: Show raw output
Nimble changes the output color and formatting when executing tasks. Some tools, such as `unittest` or  the compiler itself, use color to display information in a more readable way.

It would be nice if there was an option to show raw output (without buffering it, adding indentation or changing color). This may also be useful in order to write tasks whose output needs to be predictable - maybe because it is piped in other commands.

### PR DESCRIPTION
**Pull request by andreaferretti**
It seems to me that this should fix #305, which is the main obstacle I find in using nimble as a task runner

### CODE REVIEW

File: tests/tester.nim [hunk 1]
```diff
@@ -188,9 +188,9 @@ test "can use nimscript's setCommand":
 test "can use nimscript's setCommand with flags":
   cd "nimscript":
     let (output, exitCode) = execNimble("--debug", "cr")
-    let lines = output.strip.splitLines()
+    # let lines = output.strip.splitLines()
```
Comment (dom96) [inline tests/tester.nim]: Why was this commented out?

**Comment by andreaferretti** [comment]
Because now the output is live, hence it is not captured in your test :-)

**Comment by dom96** [comment]
huh? Why not? Why does it being live prevent it being captured?

**Comment by andreaferretti** [comment]
Umm, I will check better and figure out, on a second thought I think you
are right

**Comment by dom96** [comment]
I think the order of the messages is simply different. All you need to do is use the ``inLines`` procedure defined at the top of the tester (other tests use it too, and probably all of them should use it).

**Comment by dom96** [comment]
Thanks! btw you don't need to write "fixes nim-lang#123" in the commit message, "fixes #123" is enough.

**Comment by andreaferretti** [comment]
You are right, it was just a matter of using `inLines` :-)

**Pull request merged**